x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/additional_section/components/calendars/description.tsx

```diff
@@ -18,16 +21,28 @@ export const Description: FC = memo(({ children }) => {
   );
   return (
     <EuiDescribedFormGroup
+      fullWidth
       idAria="description"
       title={<h3>{title}</h3>}
       description={
         <FormattedMessage
           id="xpack.ml.newJob.wizard.jobDetailsStep.additionalSection.calendarsSelection.description"
-          defaultMessage="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam."
+          defaultMessage="Calendars contain a list of scheduled events for which you do not want to generate anomalies,
```

Comment:
Suggest cutting this down to `Contain a list of scheduled events for which you do not want to generate anomalies, such as planned system outages or public holidays.` with the Learn more link left as-is.

Reply:
Fixed in https://github.com/elastic/kibana/pull/51281/commits/8438634bf5c7f0afd22b6c9f71756514bad0beda

Reply:
Sorry, can you change this again to `Contains a list of scheduled events that you want to ignore, such as planned system outages or public holidays.`

Reply:
Changed in https://github.com/elastic/kibana/pull/51281/commits/045da76d02bad3df69cc17a7b5ee3bc623e7083b